app/routes/web.py

```
@@ -0,0 +1,212 @@
+        password = form.password.data
+
+        # Check if the username is already taken
+        existing_user = User.query.filter_by(username=username).first()
+        if existing_user:
+            flash("Username already exists.", "danger")
+        else:
+            # Check if this is the first user, if so, make them an admin
+            if User.query.count() == 0:
+                role = "admin"
+            else:
+                role = "user"
```

Ruff (0.8.2)

Use ternary operator `role = 'admin' if User.query.count() == 0 else 'user'` instead of `if`-`else`-block

Replace `if`-`else`-block with `role = 'admin' if User.query.count() == 0 else 'user'`

(SIM108)